docs/integrations/aws_codebuild.md

```diff
@@ -42,15 +40,43 @@ Running streampipe in CodeBuild uses the same `run-as: codebuild-user` as the in
     run-as: codebuild-user
     commands:
       - ./steampipe --version # for debugging
-      - export STEAMPIPE_WORKSPACE_CHDIR=`pwd`/steampipe-mod-terraform-aws-compliance
+      - export STEAMPIPE_MOD_LOCATION=`pwd`/steampipe-mod-terraform-aws-compliance
       - cd terraform ; ../steampipe check all --output html > steampipe_report.html
 
 ```
 
 Because the `steampipe check all` command returns the number of violations and we want CodeBuild to exit cleanly, we add `on-failure: CONTINUE` to the build phase.
 
-For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_WORKSPACE_CHDIR`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+For the actual check, we just need to tell steampipe where to find the mod `export STEAMPIPE_MOD_LOCATION`, then we change directory to the terraform and run the  `steampipe check all` command. Since steampipe was installed in the parent directory of terraform, we call it as `../steampipe`.
+
+
+## Using Steampipe Cloud
+
+Steampipe can also push snapshots to Steampipe Cloud via AWS CodeBuild. To do this we make a few changes to the buildspec file.
```

Comment:
Maybe link to https://steampipe.io/docs/cloud/overview the first time we mention Steampipe Cloud?  I would maybe add a link for 'push snapshots' to either https://steampipe.io/docs/snapshots/overview or https://steampipe.io/docs/snapshots/batch-snapshots to provide a little more context